packages/starlight/components/HeadSEO.astro

```diff
@@ -24,7 +26,7 @@ const headDefaults: z.input<ReturnType<typeof HeadConfigSchema>> = [
 		tag: 'meta',
 		attrs: { name: 'viewport', content: 'width=device-width, initial-scale=1' },
 	},
-	{ tag: 'title', content: title },
+	{ tag: 'title', content: `${pageTitle} ${titleDelimiter} ${title}` },
```

Comment:
I don't have much experience with RTL languages. What could be different? Maybe the order?

Reply:
I can do a test then. Yeah, basically that in RTL the reading order would be reversed. So should check that this template displays correctly as `title | pageTitle` rather than the opposite.

Reply:
OK, did a test and this does seem to work as expected 👍